external-crates/move/move-stdlib/sources/type_name.move

```diff
@@ -13,9 +13,20 @@ module std::type_name {
         name: String
     }
 
-    /// Return a value representation of the type `T`.
+    /// Return a value representation of the type `T`.  Package IDs
+    /// that appear in fully qualified type names in the output from
+    /// this function are original IDs (the ID of the first version of
+    /// the package, even if the type in question was introduced in a
+    /// later upgrade).
     public native fun get<T>(): TypeName;
 
+    /// Return a value representation of the type `T`.  Package IDs
+    /// that appear in fully qualified type names in the output from
+    /// this function are original IDs (the ID of the first version of
+    /// the package, even if the type in question was introduced in a
+    /// later upgrade).
+    public native fun get_original<T>(): TypeName;
```

Comment:
Thoughts on the naming here? Maybe `get_with_original_package_id`?
It's a bit of a mouthful I know. But `get` vs `get_original` feels maybe a bit off as the `original` here is not describing the `TypeName` itself, right?

Reply:
@damirka and I went back and forth on names, and couldn't come up with an ideal name, but how about `get_with_original_ids` (plural because there may be multiple in the case of a generic type instantiation) as a trade-off on precision and succinctness?

Reply:
Works for me! 

(FWIW I never really liked the name `get` either so overall I don't care too much :P)